sup3r/preprocessing/batch_handling.py

```diff
@@ -90,6 +112,71 @@ def reduce_features(high_res, output_features_ind=None):
         else:
             return high_res[..., output_features_ind]
 
+    # pylint: disable=W0613
+    @staticmethod
```

Comment:
Im confused by this method. Why do we need this for just a high_res pass-through?